public/content/documentation/web/page-level/form.md

```diff
@@ -1,80 +1,603 @@
+      Preferred contact method
+    </legend>
+  
+    <input type="radio" name="method" id="contact-email" checked>
+    <label for="contact-email">Email</label>
+  
+    <input type="radio" name="method" id="contact-sms">
+    <label for="contact-sms">SMS text</label>
+  
+    <input type="radio" name="method" id="contact-phone">
+    <label for="contact-phone">Phone</label>
+  </fieldset>
+
+  <fieldset>
+    <legend>
+      Your information
+    </legend>
+
+    <label for="email">
+      Email address
+    </label>
+    <input id="email"
+           type="email"
+           autocomplete="email"
+           spellcheck="false"
+           aria-describedby="hint-email">
+    <div class="hint" id="hint-email">
+      We’ll never sell or share your information
+    </div>
+
+    <label for="phone">
+      Phone number
+    </label>
+    <input  type="tel"
+            id="phone"
+            inputmode="numeric"
+            autocomplete="tel"
+            aria-describedby="hint-phone">
+    <div class="hint" id="hint-phone">
+      Format: 573-268-9692
+    </div>
+
+    <label for="message">
+      Your message
+    </label>
+    <textarea id="message"></textarea> 
+      
+
+  </fieldset>
+</form>
+```
+<example>
+    <form aria-label="Contact us">
+          <fieldset>
+            <legend>
+              Preferred contact method
+            </legend>
+            <input type="radio" name="method" id="contact-email" checked>
+            <label for="contact-email">Email</label>
+            <input type="radio" name="method" id="contact-sms">
+            <label for="contact-sms">SMS text</label>
+            <input type="radio" name="method" id="contact-phone">
+            <label for="contact-phone">Phone</label>
+          </fieldset>
+          <fieldset>
+            <legend>
+              Your information
+            </legend>
+            <label for="email">
+              Email address
+            </label>
+            <input id="email"
+                   type="email"
+                   autocomplete="email"
+                   spellcheck="false"
+                   aria-describedby="hint-email">
+            <div class="hint" id="hint-email">
+              We’ll never sell or share your information
+            </div>
+            <label for="phone">
+              Phone number
+            </label>
+            <input  type="tel"
+                    id="phone"
+                    inputmode="numeric"
+                    autocomplete="tel"
+                    aria-describedby="hint-phone">
+            <div class="hint" id="hint-phone">
+              Format: 573-268-9692
+            </div>
+            <label for="message">
+              Your message
+            </label>
+            <textarea id="message"></textarea>
+          </fieldset>
+        </form>
+</example>
+
+## Output from inputs
+- Screenreader support varies output can be used for a dynamic content that changes based on user inputs (example: a calculator).
```

Comment:
This bullet point doesn't read clearly to me. Should the first bullet be broken into two? Is there a more clear way of stating it? I'm not sure, but could suggest either:
- Screenreader support may vary
- When user inputs change otherwise dynamic content, consider using `output` (Ex: calculator).
OR
- Screenreader support varies when using `output` for dynamic content that chnages based on user inputs (Ex: a calculator).